### PR TITLE
[MRG] return peak amp in get_peak()

### LINKED ISSUE #4575: No amplitude from evoked.get_peak()
Wouldn't it be logical for `evoked.get_peak()` to also return the max amplitude? 
I coded this myself into a script that uses `get_peak()`, but I had to:

-arrange a new data matrix and channel names according to the pair grads logic
-find the index of the channel with maximum amplitude

`get_peak()` would already do these for free. However returning the amplitude would break API, as a 3-tuple of outputs would be needed.

### PR DESCRIPTION
**Pull request by jjnurminen**
Closes #4575

### CODE REVIEW

**Comment by agramfort** [comment]
looks good but see why CIs complain

**Comment by jjnurminen** [comment]
I forgot to fix one call to the private _get_peak() whose call signature changed

**Comment by jjnurminen** [comment]
Travis build stalled, is it related to the PR or not?

**Comment by agramfort** [comment]
ok I restarted it

**Comment by jjnurminen** [comment]
still doesn't run...

**Comment by jjnurminen** [comment]
Hooray. Someone please review + merge.

File: mne/evoked.py [hunk 1]
```diff
@@ -561,6 +562,8 @@ def get_peak(self, ch_type=None, tmin=None, tmax=None,
             Whether to return the time index instead of the latency in seconds.
         merge_grads : bool
             If True, compute peak from merged gradiometer data.
+        return_amplitude : bool
+            If True, return also the amplitude at the maximum response.
```
Comment (larsoner) [inline mne/evoked.py]: newline, `.. versionadded:: 0.16` (same with new return value below)

**Comment by jjnurminen** [comment]
There you go.

File: mne/evoked.py [hunk 2]
```diff
@@ -561,14 +562,22 @@ def get_peak(self, ch_type=None, tmin=None, tmax=None,
...
         ch_name : str
             The channel exhibiting the maximum response.
         latency : float | int
             The time point of the maximum response, either latency in seconds
             or index.
+        amplitude : float
+            The amplitude of the maximum response. Only returned if
+            return_amplitude is True.
+        
+        .. versionadded:: 0.16
```
Comment (larsoner) [inline mne/evoked.py]: needs to be indented to match the rest of the `amplitude` description

File: mne/evoked.py [hunk 3]
```diff
@@ -561,14 +562,22 @@ def get_peak(self, ch_type=None, tmin=None, tmax=None,
             Whether to return the time index instead of the latency in seconds.
         merge_grads : bool
             If True, compute peak from merged gradiometer data.
+        return_amplitude : bool
+            If True, return also the amplitude at the maximum response.
 
+        .. versionadded:: 0.16
```
Comment (larsoner) [inline mne/evoked.py]: needs to be indented (same level as the rest of the `return_amplitude` description), and needs a newline after it (it broke the `Returns` section)

https://ci.appveyor.com/project/mne-tools/mne-python/build/1.0.1146#L472

**Comment by larsoner** [comment]
> Live and learn :)

Yes, the many peculiarities of Sphinx/numpydoc/ReST ...

**Comment by larsoner** [comment]
All better https://6333-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.FVJWtJk/html/generated/mne.Evoked.html#mne.Evoked.get_peak

+1 for merge once CIs are happy

**Comment by agramfort** [comment]
fine with me

**Comment by larsoner** [comment]
Thanks @jjnurminen

**Pull request merged**